test-types/lifecycle-events.test.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * @type ./empty.d.ts
+ */
+
+import {
+  expectType
+} from 'tsd';
+
+expectType<any>(beforeAll(() => {}));
```

Comment:
Testing for any is not really testing for return types. Are we able to test for what it actually returns?

Reply:
These APIs itself have return type as `any`, and so we have used the same.

Reply:
that should be fixed - they are `void` in reality